### ISSUE TITLE
Option to turn off fog of war

### ISSUE THREAD

**Issue by kostek00** [OpenRA/OpenRA#14912]
It would be nice if you could add option to turn off fog of war in games that originally didn't had them. It's more towards players that would like play game as originally it can be.

**Comment by GraionDilach** [comment]
You can already turn it off in the lobby menu.

**Comment by kostek00** [comment]
Can you tell me where in Dune 2000 I can turn off fog of war? I don't see anywhere such option.

**Comment by reaperrr** [comment]
In skirmish/multiplayer, it's one of the checkboxes when you chose Options.
In missions it's indeed currently not possible to turn it off, so if that's what you meant, you made a valid point here.

**Comment by kostek00** [comment]
Yes, I meant campaign. Sorry, I didn't specified but i thought it's on everywhere. I didn't played skimrish or multiplayer yet.

**Comment by Luke44** [comment]
SO, does someone have an answer for this? I want to know too!
C&C Maps (ESPECIALLY CAMPAIGN) shouldn't have the regenerating fog of war.

**Comment by Luke44** [comment]
And you should have been able to know the question was about campaign missions by the wording.
“in games that originally didn't had them” Duh.
Seems gamers today got skirmish on the brain.

**Comment by Luke44** [comment]
Why did you swap Question for Feature?
What is Feature anyway?
It's a very valid question still today, especially because it hasn't been answered!!!

**Comment by abcdefg30** [comment]
It has the Feature label because we didn't add this feature to the mission browser yet, you can't disable FoW in the latest release.

**Comment by Luke44** [comment]
Cool.
Not even in Lua script?
Often scripts have a way.

**Comment by kostek00** [comment]
To begin with it wasn't question but feature request. Changing question label to feature was correct.

**Comment by Robouste** [comment]
So, is ther any plan to implement this? Original game didn't have FoW if I recall correctly.

**Comment by GodOfTwigs** [comment]
I found a solution to this problem, i will put it down in easy steps to follow.
1. you must have a code editor of your choice (can even be notepad)
2. locate your openRA folder
3. open mods folder
4. open d2k folder
5. open rules folder
6. open campaign-rules.yaml
7. on line 8 you should see: "FogCheckboxEnabled: True", change this to "FogCheckboxEnabled: False" and leave the rest untouched
8. save it (might require you to give ourself admin access to the folder)

Hope this is clear and easy to follow!

(edit: making it easier to read)

**Comment by gitflo2000** [comment]
The mentioned Settings does not work :-(
my campaign-rules.yaml does look like that:
`Player:
-ConquestVictoryConditions:
MissionObjectives:
EarlyGameOver: true
GameOverDelay: 3000

Shroud:
FogCheckboxVisible: **False**
FogCheckboxLocked: **False**
FogCheckboxEnabled: True
ExploredMapCheckboxVisible: False
ExploredMapCheckboxLocked: False
ExploredMapCheckboxEnabled: False`

**Comment by GodOfTwigs** [comment]
Hey,
I see you have a prolem in the "Shroud"
it should look like this Player:
	-ConquestVictoryConditions:
	MissionObjectives:
		EarlyGameOver: true
		GameOverDelay: 3000
	Shroud:
		FogCheckboxLocked: True
		FogCheckboxEnabled: False
		ExploredMapCheckboxLocked: True
		ExploredMapCheckboxEnabled: False

**Comment by Luke44** [comment]
> Hey, I see you have a prolem in the "Shroud" it should look like this Player: -ConquestVictoryConditions: MissionObjectives: EarlyGameOver: true GameOverDelay: 3000 Shroud: FogCheckboxLocked: True FogCheckboxEnabled: False ExploredMapCheckboxLocked: True ExploredMapCheckboxEnabled: False

The changes:

• Removed empty line above 'Shroud' item.
• Removed both '~CheckboxVisible' items.
• Set both '~CheckboxLocked' items to 'True'.

'Shroud' is unchanged.
